runtime/common/src/migrations.rs

```diff
@@ -896,6 +896,51 @@ where
     }
 }
 
+pub struct ExternalValidatorsInitialMigration<Runtime>(pub PhantomData<Runtime>);
```

Comment:
Can we add an integrqation test for this?

Reply:
Done

Reply:
*fixed*